--- a/config/common.mk
+++ b/config/common.mk
@@ -255,5 +255,7 @@ LINEAGE_VERSION := ProjectSakura-$(PRODUCT_VERSION_MAJOR).$(PRODUCT_VERSION_MINO
 LINEAGE_DISPLAY_VERSION := ProjectSakura-$(PRODUCT_VERSION_MAJOR).$(PRODUCT_VERSION_MINOR)-$(LINEAGE_BUILD)-$(SAKURA_BUILD)
 SAKURA_VERSION := $(LINEAGE_VERSION)

+-include vendor/polar/config/config.mk
+
 -include $(WORKSPACE)/build_env/image-auto-bits.mk
--include vendor/lineage/config/partner_gms.mk
+#-include vendor/lineage/config/partner_gms.mk
